server/registry/actions_artifacts.go

```diff
@@ -128,6 +132,9 @@ func (s *RegistryServer) DeleteArtifact(ctx context.Context, req *rpc.DeleteArti
 	if err != nil {
 		return nil, status.Error(codes.InvalidArgument, err.Error())
 	}
+	if system.Reserved(name) {
+		return nil, status.Error(codes.InvalidArgument, err.Error())
```

Comment:
This needs to create or use a different error than the nil one left over from the parseArtifact() call above it.

Reply:
fixed, thanks